tensorboard/webapp/metrics/views/card_renderer/scalar_card_data_table.ts

```diff
@@ -216,3 +216,12 @@ function getSortableValue(value: number | string | undefined | null) {
   }
   return value;
 }
+
+function getSortableValue(point: SelectedStepRunData, header: ColumnHeaders) {
+  switch (header) {
+    case ColumnHeaders.RUN:
+      return makeValueSortable(point[ColumnHeaders.DISPLAY_NAME]);
```

Comment:
I don't think point[ColumnHeaders.DISPLAY_NAME] is populated at this point. Verified with a plain `tensorboard` that I can't sort by RUN.

Reply:
Looks fixed now, thanks!